MaterialLib/SolidModels/Ehlers.h

```diff
@@ -41,6 +41,61 @@ namespace Solids
 {
 namespace Ehlers
 {
+//
+// Variables specific to the material model.
+//
+struct MaterialProperties
+{
+    using P = ProcessLib::Parameter<double>;
+
+    /// material parameters in relation to Ehler's single-surface model
+    /// see Ehler's paper "A single-surface yield function for geomaterials"
+    /// for more details.
+    MaterialProperties(P const& G_, P const& K_, P const& alpha_,
+                       P const& beta_, P const& gamma_, P const& delta_,
+                       P const& epsilon_, P const& m_, P const& alpha_p_,
+                       P const& beta_p_, P const& gamma_p_, P const& delta_p_,
+                       P const& epsilon_p_, P const& m_p_, P const& kappa_,
+                       P const& hardening_coefficient_)
+        : G(G_),
+          K(K_),
+          alpha(alpha_),
+          beta(beta_),
+          gamma(gamma_),
+          delta(delta_),
+          epsilon(epsilon_),
+          m(m_),
+          alpha_p(alpha_p_),
+          beta_p(beta_p_),
+          gamma_p(gamma_p_),
+          delta_p(delta_p_),
+          epsilon_p(epsilon_p_),
+          m_p(m_p_),
+          kappa(kappa_),
+          hardening_coefficient(hardening_coefficient_)
+    {
+    }
+    // basic material parameters
+    P const& G;  ///< shear modulus
+    P const& K;  ///< bulk modulus
+
+    P const& alpha;
```

Comment:
@xingyuanmiao Can you please provide the documentation for these parameters? This is also missing in the doxygen parameter documentation.